R/dependecies.R

```diff
@@ -0,0 +1,510 @@
+#' Get dependencies for all elements in lock
+get_ordered_dependencies <- function(lock) {
+   cat(crayon::blue(paste("Retrieving dependency info...")))
+   deps <- get_dependencies_for_list(lock, lock, list())
+   cat("\n")
+   deps
+}
+
+#' Recursive function to take a list and lock and extract dependencies, sorting
+#' along the way using the combine_dependencies function.
+get_dependencies_for_list <- function(master_list, lock, previously_parsed_deps) {
```

Comment:
Can you add Roxygen documentation or make it more clear what a `master_list` is?